external/storm-kafka-client/src/main/java/org/apache/storm/kafka/trident/TridentKafkaState.java

```diff
@@ -98,14 +107,19 @@ public void updateState(List<TridentTuple> tuples, TridentCollector collector) {
             }
 
             if(exceptions.size() > 0){
-                String errorMsg = "Could not retrieve result for messages " + tuples + " from topic = " + topic 
-                        + " because of the following exceptions: \n";
-                for (ExecutionException exception : exceptions) {
-                    errorMsg = errorMsg + exception.getMessage() + "\n";
-                }
-                LOG.error(errorMsg);
-                throw new FailedException(errorMsg);
-            }
+		StringBuilder errorMsg = new StringBuilder("Could not retrieve result for messages ").append(tuples)
```

Comment:
Java auto converts string concats to StringBuilder. There is no need for this. See discussion here for when is useful to use StringBuilder
http://stackoverflow.com/questions/11942368/why-use-stringbuilder-explicitly-if-the-compiler-converts-string-concatenation-t